tools/bt2img.py

```diff
@@ -37,8 +37,6 @@
     "Subtree",
     "Timeout",
     "RateController",
-    "DistanceController",
```

Comment:
Why delete these?

Reply:
Oh, I didn‘t delete them on purpose.

Reply:
We can remove RateController, DistanceController, and SpeedController from this list